concent_api/core/tests/test_integration_accept_or_reject_results.py

```diff
@@ -1675,12 +1679,13 @@ def test_requestor_send_again_subtask_results_accepted_or_rejected_when_message_
 
         task_to_compute = deserialized_force_subtask_results.ack_report_computed_task.report_computed_task.task_to_compute  # pylint: disable=no-member,
 
-        subtask = store_or_update_subtask(
+        subtask = store_subtask(
             task_id=task_to_compute.compute_task_def['task_id'],
             subtask_id=task_to_compute.compute_task_def['subtask_id'],
             provider_public_key=self.PROVIDER_PUBLIC_KEY,
             requestor_public_key=self.REQUESTOR_PUBLIC_KEY,
             state=Subtask.SubtaskState.ACCEPTED,
+            next_deadline=None,
```

Comment:
I think better than adding `next_deadline=None` everywhere where you need this is just add default value for `next_deadline` in function.

Reply:
generally ok.. but I think in this case is not good idea.
1. This function saves object to database. I think all fields should be given explicitly, not by default.
2. In production code `None` is never passed, it is not good default value. None is only passed in tests. If you insist to change it... but i would leave it as it is now.